src/com/spikes2212/gui/DashBoardController.java

```diff
@@ -0,0 +1,69 @@
+package com.spikes2212.gui;
```

Comment:
if you're only using one file, you should probably pack it in utils, not its own package